README.md

```diff
@@ -5,3 +5,16 @@ Currently in development.
 ## Installing from source
 `pip3 install .`
 
+## MacOS
+### Measuring time with oiejq
+Not available
+### Measuring time with time command
+Install those packages with brew: \
+```brew install coreutils gnu-time```
+
+## Linux
+### Measuring time with oiejq
+Download oiejq from [here](https://oij.edu.pl/zawodnik/srodowisko/oiejq.tar.gz) and add oiejq.sh to `$PATH`
```

Comment:
Those steps are not true anymore, right?

Reply:
I don’t think that in #1 I added automatic oiejq download

Reply:
Oh, I see, sorry. Then I hope someday you'll create a PR with it :)